src/main/java/com/ncov/module/controller/request/material/MaterialRequest.java

```diff
@@ -24,6 +24,8 @@
     @ApiModelProperty(value = "发布/供货机构名称")
     @NotEmpty(message = "机构名称不能为空")
     private String organisationName;
+    @ApiModelProperty(value = "发布/供货机构Logo")
```

Comment:
No need in request since the logo is saved upon registration, not during publishing